docs/src/main/sphinx/release/release-397.md

```diff
@@ -48,8 +48,8 @@
 * Reduce query latency when using a Glue catalog for metadata. ({issue}`13875`)
 * Fix materalized views temporarily appearing empty when a refresh is about to
   complete. ({issue}`14145`)
-* Fix potential table corruption when changing a table before committing to the
-  Hive metastore has completed. ({issue}`14118`)
+* Fix potential table corruption when changing a table before it is known if
+  committing to the Hive metastore has failed or succeeded. ({issue}`14174`)
```

Comment:
Thanks for responding to https://github.com/trinodb/trino/pull/14194#discussion_r981289095

I still don't think the wording conveys the meaning. "changing a table before" sounds like there is some kind of a race condition in play, where it's not the case